Gems/ScriptCanvas/Code/Editor/Components/EditorGraph.cpp

```diff
@@ -100,20 +100,8 @@ namespace EditorGraphCpp
 }
 namespace ScriptCanvasEditor
 {
-    namespace EditorGraph
-    {
-        static const char* GetMimeType()
-        {
-            return "application/x-o3de-scriptcanvas";
-        }
-
-        static const char* GetWrappedNodeGroupingMimeType()
-        {
-            return "application/x-03de-scriptcanvas-wrappednodegrouping";
-        }
-    }
```

Comment:
Is there a reason this was moved from the .cpp file to the header?

Reply:
its in the same namespace as the class EditorGraph.

Reply:
Well it looks like you added it as a static member function to the `EditorGraph` class, so the implementation can still be in the .cpp file, It just would just need to have `EditorGraph::` prepended to the function name

Reply:
you mean move the declaration into the cpp file?